stable/gocd/values.yaml

```diff
@@ -246,6 +246,10 @@ agent:
   # preStop:
   #  - "/bin/bash"
   #  - "/disable_and_stop.sh"
+  # agent.postStart - array of commands to use in agent post-start lifecycle hook
```

Comment:
Is it really an array of commands? Or, one command with arguments? I feel that either the comment should change or the examples could be changed.

Reply:
Indeed, it's a command with arguments. Updated the comments for both preStop and postStart.